common4j/src/main/com/microsoft/identity/common/java/controllers/BaseController.java

```diff
@@ -857,6 +857,7 @@ protected TokenResult performSilentTokenRequest(
             ((MicrosoftTokenRequest) refreshTokenRequest).setClaims(parameters.getClaimsRequestJson());
             ((MicrosoftTokenRequest) refreshTokenRequest).setClientAppName(parameters.getApplicationName());
             ((MicrosoftTokenRequest) refreshTokenRequest).setClientAppVersion(parameters.getApplicationVersion());
+            ((MicrosoftTokenRequest) refreshTokenRequest).setClientAppVersion(parameters.getCorrelationId());
```

Comment:
@SammyO @shahzaibj @fadidurah Discussion from the old PR in below screenshot.
![image](https://github.com/AzureAD/microsoft-authentication-library-common-for-android/assets/106561208/7d93fde8-2def-473f-8f2d-4b06b5160064)